src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
```

Comment:
The path to the star image is absolute. It's better to use relative paths for assets in CSS. This will make your code more portable, because it won't rely on the project's absolute structure.